Solutions/IronNet IronDefense/Playbooks/IronNet_UpdateIronDefenseAlerts/azuredeploy.json

```diff
@@ -98,73 +105,81 @@
 							"defaultValue": {},
 							"type": "Object"
 						},
-						"IronNetUrl": {
-							"defaultValue": "[parameters('IronDefenseUrl')]",
-							"type": "SecureString"
+						"tenant_id": {
+							"defaultValue": "[subscription().tenantId]",
+							"type": "String"
 						},
-						"Password": {
-							"defaultValue": "[parameters('IronApiPassword')]",
-							"type": "SecureString"
+						"irondefense_url": {
+							"defaultValue": "[parameters('playbook1-IronDefenseUrl')]",
```

Comment:
Please remove playbook1 from all the "defaultvalue" section
Paramters used in "defaultvalue" should be matching with the parameters